src/launch/launch-box.html

```diff
@@ -2,7 +2,7 @@
   <div class="launch-box-container">
     <div class="topSection">
       <div class="heading heading3 title">${name}</div>
-      <div class="subtext">${subTitle1}</div>
+      <div class="subtext" innerHtml.bind="subTitle1"></div>
```

Comment:
why did you do this?

Reply:
@dkent600 because subTitle1 contains html a href tag for the link